tests/contrib/cassandra/test.py

```diff
@@ -21,6 +20,22 @@
 logging.getLogger('cassandra').setLevel(logging.INFO)
 
 
+def setUpModule():
```

Comment:
Speeds up a lot our tests; each `DROP` costs about 10-20 seconds (per test) so doing that only once is really faster.